modules/fedimint-mint-client/src/lib.rs

```diff
@@ -317,6 +317,11 @@ impl FromStr for OOBNotes {
     /// Decode a set of out-of-band e-cash notes from a base64 string.
     fn from_str(s: &str) -> Result<Self, Self::Err> {
         let s: String = s.chars().filter(|&c| !c.is_whitespace()).collect();
+
+        if let Ok(notes_v2) = OOBNotesV2::decode_base64(&s) {
+            return notes_v2.into_v1();
+        }
+
```

Comment:
This breaks en/decoding round-trips, but that's ok I guess. We might want to separate the encoding structs from the semantic structs eventually (e.g. have v1 and v2 structs that can both be converted from/to a common notes struct).